src/main/resources/org/broadinstitute/hellbender/utils/plotter/AlleleFractionResultsPlotting.R

```diff
@@ -0,0 +1,126 @@
+## HG19 chromosome lengths and centromere bounds relative to beginning of chromosomes
+chromosome_lengths = c(249250621, 243199373, 198022430, 191154276, 180915260, 171115067, 159138663,
+        146364022, 141213431, 135534747, 135006516, 133851895, 115169878, 107349540,
+        102531392, 90354753, 81195210, 78077248, 59128983, 63025520, 48129895, 51304566)
+centromere_starts = c(121535434, 92326171, 90504854, 49660117, 46405641, 58830166, 58054331, 43838887, 47367679,
+	39254935, 51644205, 34856694, 16000000, 16000000, 17000000, 35335801, 22263006, 15460898, 24681782,
+	26369569, 11288129, 13000000)
+centromere_ends = c(124535434, 95326171, 93504854, 52660117, 49405641, 61830166, 61054331, 46838887, 50367679,
+	42254935, 54644205, 37856694, 19000000, 19000000, 20000000, 38335801, 25263006, 18460898, 27681782,
+	29369569, 14288129, 16000000)
+
+###Add X and Y chromosomes###
+if(sex_chrs){
+    chromosome_lengths = c(chromosome_lengths, 155270560, 59373566)
+    centromere_starts = c(centromere_starts, 58632012, 10104553)
+    centromere_ends = c(centromere_ends, 61632012, 13104553)
+}
+
+#henceforth we use genomic positions scaled relative to the total genome length
+genome_length = sum(chromosome_lengths)
+chromosome_lengths = chromosome_lengths / genome_length
+centromere_starts = centromere_starts / genome_length   #relative to chromosome starts
+centromere_ends = centromere_ends / genome_length       #relative to chromosome ends
+
+num_chromosomes = length(chromosome_lengths)
+
+chromosome_ends = cumsum(chromosome_lengths)
+chromosome_starts = c(0, head(chromosome_ends, -1))
+chromosome_centers = (chromosome_starts + chromosome_ends) / 2
+
+centromere_starts = chromosome_starts + centromere_starts
+centromere_ends = chromosome_starts + centromere_ends
+
+# Use a function for debugging purposes
+create_allele_fraction_plots_file = function(sample_name, snp_counts_file, segments_file, output_dir) {
+	# Read in file and extract needed data
+	snp_counts = read.table(snp_counts_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+	segments = read.table(segments_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+
+
+	snpContig = snp_counts[,"CONTIG"]
+    snpContig[snpContig=="X"]=23
+    snpContig[snpContig=="Y"]=24
+    snp_counts[,"CONTIG"]=as.numeric(snpContig)
+
+    segments[segments["Chromosome"]=="X","Chromosome"]=23
+    segments[segments["Chromosome"]=="Y","Chromosome"]=24
+    segments[,"Chromosome"]=as.numeric(segments[,"Chromosome"])
+
+    #exclude segments with no SNPS
+    segments = segments[segments$Num_SNPs > 0,]
+
+    plot_file_name = file.path( output_dir, paste(sample_name, "_FullGenome", ".png", sep=""))
+    png(plot_file_name, 12, 7, units="in", type="cairo", res=300, bg="white")
+    SetUpPlot("Allele Fraction", "", TRUE)
+    AlleleFractionPlot(snp_counts, segments)
+    dev.off()
+}
+
+## Chromosome Background for data
+SetUpPlot = function(y.lab, x.lab, label_chromosomes){
```

Comment:
Is there a reason for this to be different between plotting methods? I would suggest copying the tangent plotting and changing variable names on both to regularize whatever you think could use changing.